weblate/api/views.py

```diff
@@ -631,6 +631,35 @@ def delete_components(self, request, id, component_id):
         obj.components.remove(component)
         return Response(status=HTTP_204_NO_CONTENT)
 
+    @action(detail=True, methods=["post"], url_path="admins")
+    def grant_admin(self, request, id):
+        group = self.get_object()
+        self.perm_check(request)
```

Comment:
The perm_check should be extended to optionally accept object, so that group admins can also use this interface.